content/darkroom/masking-and-blending/blend-modes.md

```diff
@@ -5,44 +5,159 @@ weight: 20
+
+```
+final_output = (1.0 - opacity) * module_input + opacity * blended_output
+```
+
+where the `blended_output` is a combination of the input and output images, depending on the blend mode (below), and the `opacity` is defined 'per-pixel' by a combination of the mask and global opacity parameter.
+
+# normal modes
+
 normal
 : The most commonly used blend mode, "normal" simply mixes input and output to an extent determined by the opacity parameter. This mode is commonly used to reduce the strength of a module's effect by reducing the opacity. This is also usually the blend mode of choice when applying a module's effect selectively with masks.
 
 normal bounded
+: _not available in the "RGB (Scene)" color space_
 : This blend mode is the same as “normal”, except that the input and output data are clamped to a particular min/max value range. Out-of-range values are effectively blocked and are not passed to subsequent modules. Sometimes this helps to prevent artifacts. However, in most cases (e.g. highly color-saturated extreme highlights) it is better to let unbound values travel through the pixelpipe to be properly handled later. The “normal” blend mode is therefore usually preferred.
 
-lightness
-: This blend mode mixes lightness from the input and output images. Color (chroma and hue) is taken unaltered from the input image.
+# arithmetic modes
 
-chroma
-: This blend mode mixes chroma (saturation) from the input and output images. Lightness and hue are taken unaltered from the input image.
+addition
+: Add together the pixel values of the input and output images, lightening the output. When blending in the "RGB (scene)" color space, the pixel values of the output image are multiplied by a value proportional to the "blend fulcrum".
 
-hue
-: This blend mode mixes hue (color tint) from the input and output images. Lightness and chroma are taken unaltered from the input image.
+subtract
+: Subtract the pixel value of the _output_ from the _input_. When blending in the "RGB (scene)" color space, the pixel values of the output image are multiplied by a value proportional to the "blend fulcrum". Pixel values less than 0 are set to 0.
 
-color
-: This blend mode mixes color (chroma and hue) from the input and output images. Lightness is taken unaltered from the input image. 
+subtract reverse
+: _only available in the "RGB (Scene)" color space_
+: Subtract the pixel value of the _input_ from the _output_. In this case, the "blend fulcrum" parameter of the "RGB (scene)" color space is applied to the input image.
 
-: _Caution: When modules drastically modify hue (e.g. when generating complementary colors) this blend mode can result in strong color noise._
+multiply
+: Multiply the pixel values of the input and output together. As all pixel values are between 0 and 1.0, the resulting output will always be darker. When blending in the "RGB (scene)" color space, this value is further multiplied by a value proportional to the "blend fulcrum".
```

Comment:
You may add : "Multiply simulates an optical variable density filter, where the density is defined by the output of the module. It has many application, from blooming and local contrast enhancements (used with a blur or low-pass filter output) to dodging/burning and global contrast enhancements (used with exposure output). The fulcrum sets the output intensity threshold between darkening and brightening (any RGB value below fulcrum will darken)".